nixnet/_funcs.py

```diff
@@ -695,3 +695,43 @@ def nxdb_undeploy(
         database_alias_ctypes,
     )
     _errors.check_for_error(result.value)
+
+
+def nxdb_get_database_list(
+    ip_address,  # type: typing.Text
+):
+    # type: (...) -> typing.Tuple[List[typing.Text], List[typing.Text], int]
+    ip_address_ctypes = _ctypedefs.char_p(ip_address.encode('ascii'))
+    size_of_alias_buffer, size_of_filepath_buffer = nxdb_get_database_list_sizes(ip_address)
```

Comment:
I've cleaned this up and move the call the sizes function one layer above in databases.py